docs/pages/blog/hello-grid-version-2.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Hello, Grid version 2
+description: We're excited to introduce the Grid version 2 that comes with new features and improvements!
+date: 2022-08-20T00:00:00.000Z
+authors: ['siriwatknp']
+tags: ['MUI Core', 'News']
+card: false
+---
+
+We have shipped the new `Grid` component in [Material UI v5.9.0](https://github.com/mui/material-ui/releases/tag/v5.9.0) that will replace the existing one in the next major release.
+
+```js
+import Grid from '@mui/material/Grid'; // The current grid, a.k.a. Grid v1
+import Grid2 from '@mui/material/Unstable_Grid2'; // The new grid, a.k.a. Grid v2
```

Comment:
The import name depends on the developers but we could make it clear in the demos to use `Grid2`.

The snippets below produce the same result.

```js
import Grid2 from '@mui/material/Unstable_Grid2';

// the usage should be Grid2
<Grid2>...</Grid2>
```

```js
import Grid from '@mui/material/Unstable_Grid2';

// the usage should be Grid
<Grid>...</Grid>
```

Reply:
I see. We should be consistent throughout the doc, so either we call it `Grid` instead of `Grid2` in this `import`, or else we should change all instances of `<Grid>` to `<Grid2>` in the examples. I think we should just call it `Grid` everywhere for the sake of simplicity.

Reply:
will go with `Grid`.